Example/SectionedCollectionView/ViewControllers/SectionedCollectionViewController.swift

```diff
@@ -45,6 +45,9 @@ class SectionedCollectionViewController: UIViewController {
         
         sectionedCollectionView.setDataSource(sections: sections)
         sectionedCollectionView.delegate = self
+        
+        let indexs = [IndexPath(row: 0, section: 2), IndexPath(row: 0, section: 0), IndexPath(row: 2, section: 0)]
```

Comment:
change indexs -> indexes
add a comment before these lines -> \\ In this way we can select items programtically